scripts/log-capture

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-date=$(date +%F_%R)
+date=$(date +%F_%H%M%S)
```

Comment:
`%R` represents only `%H:%M`.

Reply:
I know it does but I think seconds should be included too. You can have two reports under one minute when it describes just one click difference.